lib/cssom.js

```diff
@@ -11,12 +11,26 @@
 declare class StyleSheet {
   disabled: boolean;
   href: string;
+  media: MediaList;
   ownerNode: Node;
   parentStyleSheet: ?StyleSheet;
   title: string;
   type: string;
 }
 
+declare class StyleSheetList {
+  length: number;
+  [index: number]: StyleSheet;
+}
+
+declare class MediaList {
+  mediaText: DOMString;
```

Comment:
let's just use `string` instead of `DOMString`. we do that in a lot of other places so I think it's how we're handling it (before my time, though)